lib/executor/helpers.go

```diff
@@ -72,6 +72,16 @@ func validateStages(stages []Stage) []error {
 		} else if s.Target.Int64 < 0 {
 			errors = append(errors, fmt.Errorf("the target for stage %d shouldn't be negative", stageNum))
 		}
+
+		switch s.Progression.String {
+		case "linear",
+			"constant",
```

Comment:
Did you mean "instant" instead? If not, could you clarify your reasoning? My assumption is that a `constant` interpolation would either mean a linear one (as in, "a constant rate of change"), or, no movement at all, in which case, this usage is already addressed by the existing feature-set of options (although it might feel "hacky": `stages: [{duration: '1m', target: 10}, {duration: '10m', target: 10}`, would, from my understanding, perform a "constant" rate of change).